scripts/build_example

```diff
@@ -95,15 +94,16 @@ esac
 
 if [[ -n "${docker_config}" ]]; then
   "${SCRIPTS_DIR}/build_server" -s "${docker_config}"
-  if [[ "${EXAMPLE}" == "tensorflow" ]]; then
-    bazel --output_base="${CACHE_DIR}/emscripten" build "${bazel_build_flags[@]}" --config=emscripten //examples/tensorflow/server:tensorflow_image.tar
-  else
-    bazel build "${bazel_build_flags[@]}" "--config=clang" "//examples/${EXAMPLE}/server:${EXAMPLE}_image.tar"
-  fi
+  # Build the base server.
+  docker build -t oak_docker -f ./oak/server/Dockerfile ./oak/server
+  # Build and save the example
+  docker build -t "${EXAMPLE}_image" -f ./examples/Dockerfile "./examples/${EXAMPLE}"
+  docker save "${EXAMPLE}_image" > "./examples/${EXAMPLE}/bin/${EXAMPLE}_image.tar"
```

Comment:
What is this step for?

Reply:
We first build the server image, on top of that we put the "application" and we save it so we can later "push it". I am not really sure we need the .tar step, but @ipetr0v almost convinced me in a slack chat :)

Reply:
It's actually quite interesting. We could, potentially, have 1 single image, but the issue is that the Docker context would need to be the first common parent directory of the example and server, which is oak top-most directory. And because we have a .dockerignore there for the "overall" image, that conflicts with this. So this is why I made it a two step image. It's also nicer, since we can ideally build the server image once and add to it for each example